src/panels/lovelace/cards/hui-entities-card.js

```diff
@@ -122,7 +125,11 @@ class HuiEntitiesCard extends EventsMixin(PolymerElement) {
         !DOMAINS_HIDE_MORE_INFO.includes(computeDomain(entityId))
       ) {
         element.classList.add("state-card-dialog");
-        element.addEventListener("click", () =>
+        longPressBind(element);
+        element.addEventListener("clicked", () =>
+          toggleEntity(this.hass, entityId)
+        );
+        element.addEventListener("hold", () =>
```

Comment:
we should make our custom events have `ha-` prefix so that if the platform ever adds a `hold` event, we're not screwed.

Reply:
Ooh. Good idea!
`ha-click` and `ha-hold`